test/test-functions

```diff
@@ -96,8 +97,12 @@ $KERNEL_APPEND \
 }
 
 run_nspawn() {
+    local _nspawn_cmd="../../systemd-nspawn --register=no --kill-signal=SIGKILL --directory=$TESTDIR/nspawn-root $ROOTLIBDIR/systemd $KERNEL_APPEND"
```

Comment:
Hm,
```
+    local _nspawn_cmd="../../systemd-nspawn --register=no --kill-signal=SIGKILL --directory=$TESTDIR/nspawn-root $ROOTLIBDIR/systemd $KERNEL_APPEND"
...
-    ../../systemd-nspawn --register=no --directory=$TESTDIR/nspawn-root $ROOTLIBDIR/systemd $KERNEL_APPEND
```
:)

> This hardcoded path will fail with an out-of-tree build indeed.

You can't even [run](https://github.com/systemd/systemd/blob/master/test/README.testsuite) this test with an out-of-tree build
```
$ mkdir BUILD
$ cd BUILD
$ ../autogen.sh c
$ ls -l
config.h  config.log  config.status  libtool  Makefile  po  src  stamp-h1
```

Am I missinig something?:)